packages/editor-ui/src/plugins/i18n/index.ts

```diff
@@ -230,14 +229,14 @@ export class I18nClass {
 			 * Hint for an input, whether top-level or nested.
 			 */
 			hint(
-				parameter: { name: string; hint: string; type: string },
+				parameter: INodeProperties,
 				path: string,
 			) {
 				const middleKey = deriveMiddleKey(path, parameter);
 
 				return context.dynamicRender({
 					key: `${initialKey}.${middleKey}.hint`,
-					fallback: parameter.hint,
+					fallback: parameter.hint || '',
```

Comment:
added proper typings to all of those functions and used the bullish coalescing operator (sometimes I still don't trust the types and using the ternary or operator, just in case if even the types are not matching, like: `0 || ''` will be string but not `0 ?? ''`)